defaults/main.yml

```diff
@@ -1,8 +1,4 @@
-# SPDX-License-Identifier: MIT
 ---
-# Here is the right place to put the role's input variables.
-# This file also serves as a documentation for such a variables.
-
-# Examples of role input variables:
-template_foo: "foo"
-template_bar: "bar"
+vpn_provider: libreswan
+vpn_auth_method: psk
+vpn_regen_keys: false
```

Comment:
add the default here: `vpn_connections: []`

Reply:
fixed in commit [f1ad0f7](https://github.com/linux-system-roles/vpn/pull/3/commits/f1ad0f7d34814b64354250fc1e9b459673a49d1b)